dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/mail/MailMessageCreator.java

```diff
@@ -447,15 +550,23 @@ private String getSubjectTextForDatasetVersion(NotificationType notificationType
         if (notificationType == FILESYSTEMIMPORT) {
             try {
                 List<String> dsNameAsList = Collections.singletonList(datasetVersion.getDataset().getDisplayName());
-                return BundleUtil.getStringFromBundle("notification.email.import.filesystem.subject", dsNameAsList);
+                return BundleUtil.getStringFromBundle("notification.email.import.filesystem.subject",
+                                                      getCurrentUser().getNotificationsLanguage(),
+                                                      dsNameAsList);
             } catch (Exception e) {
-                return BundleUtil.getStringFromBundle("notification.email.import.filesystem.subject", Collections.singletonList(rootDataverseName));
+                return BundleUtil.getStringFromBundle("notification.email.import.filesystem.subject",
+                                                      getCurrentUser().getNotificationsLanguage(),
+                                                      Collections.singletonList(rootDataverseName));
             }
         }
 
         return StringUtils.EMPTY;
     }
 
+    private AuthenticatedUser getCurrentUser() {
+        return (AuthenticatedUser) session.getUser();
```

Comment:
The whole point was to use language of user that will receive the email - not language of the user that initiates the action